libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -426,6 +430,14 @@ class MapboxNavigation(
         navigator.resetRideSession()
     }
 
+    /**
+     * Query if the [TripSession] is running a foreground service
```

Comment:
```suggestion
     * Query if the [MapboxNavigation] is running a foreground service
```
nit: `TripSession` is an internal API.